health/health_config.c

```diff
@@ -662,15 +662,19 @@ static int health_readfile(const char *filename, void *data) {
 
         if(hash == hash_alarm && !strcasecmp(key, HEALTH_ALARM_KEY)) {
             if(rc) {
-                if(ignore_this || !alert_hash_and_store_config(rc->config_hash_id, alert_cfg) || !rrdcalc_add_alarm_from_config(host, rc)) {
+                if (host == localhost)
```

Comment:
What happens if you have health disabled on the parent but enabled for the children?

Reply:
You are of course right. Will change.